webui/react/src/pages/TrialDetails.tsx

```diff
@@ -202,8 +227,35 @@ const TrialDetailsComp: React.FC = () => {
       });
     }
 
+    if (
+      showExperimentArtifacts &&
+      experiment.modelDefinitionSize !== 0 &&
+      isSingleTrialExperiment(experiment) &&
```

Comment:
the feedback we've got indicates that we want the code tab in the trial view regardless of the single trial status
```suggestion
```